boilerplate/app/screens/DemoComponentsScreen/DemoItem.tsx

```diff
@@ -1,21 +1,15 @@
-import React, { Fragment } from "react"
-import { ListRenderItemInfo, TextStyle, View, ViewStyle } from "react-native"
+import React from "react"
+import { TextStyle, View, ViewStyle } from "react-native"
 import { Text } from "../../components"
 import { Demo } from "./DemoComponentsScreen"
 
-export function DemoItem(props: ListRenderItemInfo<Demo>) {
-  const { item } = props
-
+export function DemoItem({ name, description }: Demo) {
   return (
     <View style={$demoItemContainer}>
       <Text preset="heading" style={$title}>
-        {item.name}
+        {name}
       </Text>
-      <Text style={$description}>{item.description}</Text>
-
-      {item.useCases.map((i) => (
-        <Fragment key={i.props.name}>{i}</Fragment>
-      ))}
```

Comment:
If we're not wrapping usecases in DemoItem anymore, it might make sense to just move the name/description texts into the sectionListHeader and delete this component so we don't have to manage an extra component here.